io/disk/usb_tests.py.data/usb_tests.yaml

```diff
@@ -0,0 +1,4 @@
+usb_disk:
```

Comment:
rename to disk, other wise it will be new param in input file

Reply:
ok